style/style.css

```diff
@@ -170,4 +169,14 @@ span.close {
 	position: absolute;
 	left: 50%;
 	transform: translate(-50%,-200%);
+}
+
+#loginButton {
```

Comment:
Prefer sticking to one format of naming CSS selectors.
Either use snake_case_like_this or camelCase.